### PR TITLE
about page UI

### PR DESCRIPTION
**Pull request by m6265m**
Implement About Page UI

### CODE REVIEW

File: app/about/page.tsx [hunk 1]
```diff
@@ -1,8 +1,35 @@
+import AboutCard from 'src/components/about/AboutCard';
 import contentJSON from 'src/constants/content.json';
 
 const About: React.FC = () => {
   const content = contentJSON.about;
-  return <div>{content.title}</div>;
+  return (
+    <div>
+      <div className='flex h-[412px] flex-col items-center justify-center bg-black px-4'>
```
Comment (Amber-Riaz) [inline app/about/page.tsx:8]: Please move height to tailwind.config.js.

File: app/about/page.tsx [hunk 2]
```diff
@@ -1,8 +1,35 @@
...
+        <h1 className='mb-3 text-center text-4xl font-bold tracking-[0.08em] text-primary-yellow md:mb-9 md:text-[64px]'>
+          {content.firstSection.title}
+        </h1>
+        <p className='w-full text-center text-base font-normal leading-[1.4em] text-white md:w-[710px] md:text-[24px]'>
```
Comment (Amber-Riaz) [inline app/about/page.tsx:12]: Move width to tailwind.config.js

File: app/about/page.tsx [hunk 3]
```diff
@@ -1,8 +1,35 @@
+import AboutCard from 'src/components/about/AboutCard';
 import contentJSON from 'src/constants/content.json';
 
 const About: React.FC = () => {
   const content = contentJSON.about;
-  return <div>{content.title}</div>;
+  return (
+    <div>
+      <div className='flex h-[412px] flex-col items-center justify-center bg-black px-4'>
+        <h1 className='mb-3 text-center text-4xl font-bold tracking-[0.08em] text-primary-yellow md:mb-9 md:text-[64px]'>
```
Comment (Amber-Riaz) [inline app/about/page.tsx:9]: Move text to tailwind .config.js

File: src/components/about/AboutCard.tsx [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+}
+
+const AboutCard: React.FC<Props> = ({ name, image, title, link }) => {
+  return (
+    <a href={link} target='_blank' rel='noreferrer'>
+      <div className='mb-10 flex flex-col'>
+        <img
+          src={image}
+          alt='employee image'
+          className='h-[412px] w-[482px] mb-10 object-cover'
```
Comment (Amber-Riaz) [inline src/components/about/AboutCard.tsx:17]: Also move these to tailwindconfig.js

Comment (m6265m) [reply src/components/about/AboutCard.tsx:17]: I will move these if I need to use it consistently throughout the app. Right now I just need these in those specific components and no where else.

**Review by Amber-Riaz — CHANGES_REQUESTED** [review]
minor changes suggested.

**Review by Amber-Riaz — APPROVED** [review]

**Pull request merged**